src/common/paths/query_suggestions.yml

```diff
@@ -2,18 +2,64 @@ get:
   operationId: GetQuerySuggestions
   tags:
     - Query
-  summary: Retrieve query suggestions
+  summary: Retrieve all query functions
+  description: |
+    Returns a list of all query functions.
+
+    Use this endpoint to retrieve a list of all the possible query functions
+    that InfluxData currently supports. The endpoint additionally exposes helper
+    functions that are not meant to be used while building a query such as
+    [`_sortLimit()`](https://github.com/influxdata/flux/blob/a768568a4cbeb299433ee31a059e599d290fcfbf/libflux/flux/benches/everything.flux#L2247)
+
+    #### Limitations
+
+    - The endpoint does not return all of the functions in the Flux standard
+    library. It only returns query functions.
+
+    - Using `/api/v2/query/suggestions/` with cURL will result in a HTTP `301
+    Moved Permanently` status code. Please use `/api/v2/query/suggestions`.
```

Comment:
This took me a second to process. Maybe call out the fact that the trailing slash is what causes the problem.